python/huggingfaceserver/health_check.py

```diff
@@ -222,32 +218,32 @@ def main():
     runtime_health_parser.add_argument("--health_check_url", help="Health check URL")
     runtime_health_parser.add_argument("--probe_name", help="Probe name")
 
-    # Check if registered node is the same as pipelineParalleSize
+    # Check if registered node is the same as rayNodeCount
     reigstered_node_parser = subparsers.add_parser(
         "registered_nodes",
-        help="Check if registered nodes are the same as pipeline parallel size",
+        help="Check if registered nodes are the same as Ray node count",
     )
     reigstered_node_parser.add_argument(
-        "--pipeline_parallel_size",
+        "--ray_node_count",
         type=int,
-        default=default_pipeline_parallel_size,
-        help="Pipeline parallel size",
+        default=default_ray_node_count,
+        help="Ray node count",
     )
     reigstered_node_parser.add_argument(
-        "--retries", type=int, default=0, help="Pipeline parallel size"
+        "--retries", type=int, default=0, help="Ray node count"
     )
     reigstered_node_parser.add_argument("--probe_name", help="Probe name")
 
-    # Check if registered node is the same as pipelineParalleSize/ runtime health subcommand
+    # Check if registered node is the same as rayNodeCount/ runtime health subcommand
     registered_node_and_runtime_health_parser = subparsers.add_parser(
         "registered_node_and_runtime_health",
         help="Check node counts and runtime health",
     )
     registered_node_and_runtime_health_parser.add_argument(
-        "--pipeline_parallel_size",
+        "--ray_node_count",
```

Comment:
Just as non-actionable feedback: I think these renamings are good as long as this feature is not advertised as stable (thus no changes requested for now).

Once we say multi-node/mult-gpu we should be more careful, as these kind of renamings will lead to re-creation of any existing pods, which sometimes is not very tolerable in production envs.

Reply:
Yes, I agree with your point.
That said, when we introduce a change to core logic like this, users typically need to restart the pod anyway in order to pick up the new environment variable. So from that perspective, the impact is somewhat inevitable.